src/Components/Books/Books.js

```diff
@@ -1,10 +1,47 @@
+/* eslint-disable camelcase */
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import Header from '../Navigation/Header';
 import Book from './Book';
 import AddBook from './AllBook';
 import { fetchBooks, removeBook } from '../../redux/books/books';
 
+// const booksList = () => {
+//   const booksList = useSelector((state) => state.books);
+//   const dispatch = useDispatch();
+
+//   useEffect(() => {
+//     dispatch(fetchBooks());
+//   }, []);
+
+//   return (
```

Comment:
- [ ] Kindly remove all pieces of commented code  so as to keep your work clean 👍